bigbluebutton-html5/imports/ui/components/video-provider/service.js

```diff
@@ -76,7 +76,6 @@ class VideoService {
   getAllWebcamUsers() {
     const webcamsLocked = this.webcamsLocked();
     const webcamsOnlyForModerator = this.webcamsOnlyForModerator();
-    const hideUserList = this.hideUserList();
```

Comment:
You're removing all references to `hideUserList` - you would need to remove the function itself too.